File: csharp-snippets/Main/Addresses.cs
using System.Collections.Generic;

namespace Richargh.BillionDollar.Main
{
    public class Addresses
    {
        private readonly Dictionary<PersonId, Address> _addresses = new();
        
        public Address FindAddress(PersonId personId)
        {
            return _addresses[personId];
        }
    }
}

File: java-rent-nullaway/src/main/java/de/richargh/billiondollar/people/internal/Addresses.java
package de.richargh.billiondollar.people.internal;

import de.richargh.billiondollar.people.exposed.Address;
import de.richargh.billiondollar.people.exposed.EmployeeId;

import java.util.Optional;

public interface Addresses {

    long count();

    void put(EmployeeId id, Address address);

    Optional<Address> getById(EmployeeId employeeId);
}
